perfiles_usuario/utils.py

```diff
@@ -0,0 +1,105 @@
+""" Utility functions and constants that will be used in the project.
+"""
+
+# Names of groups for users
+ADMINISTRADOR_GROUP = 'administrador'
+CAPTURISTA_GROUP = 'capturista'
+DIRECTIVO_GROUP = 'directivo'
+SERVICIOS_ESCOLARES_GROUP = 'servicios_escolares'
+
+
+def is_member(user, groups):
+    """ Test if a user belongs to any of the groups provided.
+
+    This function is meant to be used by the user_passes_test decorator to control access
+    to views.
+
+    Parameters
+    ----------
+    user : django.contrib.auth.models.User
+        The user which we are trying to identify that belongs to a certain group
```

Comment:
Use dots at the end of every sentence, in order to keep consistency within the docs.